### DISCUSSION TITLE
Menu Feature - couple of unimportant questions

### DISCUSSION THREAD

**Discussion by scleaver** [OrchardCMS/OrchardCore#8586] [General]
Hey people, couple of tidying up things in the menu feature. Not a biggy but I feel that ContentMenuItem and LinkMenuItem shouldn't be attachable by default. It's just that they appear in the Parts list when you make a Content Type and it feels wrong.

Also there is a part in the recipe called "MenuItemPart" but it doesn't appear to have any function. Is this a stray Part?

**Comment by Skrypt** [comment]
Good point. Have you tried removing the attachable setting on them? Would need to test to see if you can add a ContentMenuItem or LinkMenuItem to a menu afterward. Just to make sure it doesn't break the Menu feature. Else, I would say you are right.
